hadoop-tools/hadoop-aws/src/test/java/org/apache/hadoop/fs/s3a/ITestS3APrefetchingInputStream.java

```diff
@@ -294,4 +294,34 @@ public void testStatusProbesAfterClosingStream() throws Throwable {
 
   }
 
+  @Test
+  public void testCacheFileExistence() throws Throwable {
```

Comment:
is there any cleanup of these files after? i know a maven clean will do it, but these may be be big files and create problems on docker container runs. best to do an rm of the dir, or at least the files, afterwards

Reply:
Sounds reasonable to let the test clean it up rather than rely on mvn cleanup. Done.

Reply:
Sorry, I had to remove this change in the latest revision.

The problem could arise with the file cleanup: we would not know which exact cache file was created by this test, meaning what if another test (with -Dprefetch option) running in parallel also creates a new cache file and we remove it as part of this test? That can screw up parallel tests.

Reply:
Update: this is now taken care of.